src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -343,17 +343,20 @@ private static String getCmdOutput(File workDir, String... cmd) throws Exception
     private static int runTestCommand(TestCommandArgs testArgs, JCommander jc) {
         // Figure out module name.
         // Join together spaced out names with underscores if necessary.
+        int returnCode = 131;
```

Comment:
FWIW, the system returns 2 when unit tests fail. But 2 represents "No such file or directory"...